### PR TITLE
Update implementation of SleepSetupView

### LINKED ISSUE #71: Update implementation of the SleepSetupView
The part of #69 

In scope of #23 we have successfully implemented the `SleepSetupView`, but unfortunately it doesn't match the [provided mockups](https://www.figma.com/file/Q3Vq556r3LTQrOokAggk1Z/Sleep-Magic-App-Design?node-id=0%3A1&t=9Jf97CaFnqWDDYri-1).

We have to update the implementation to match the mockup (except the fonts as we have an agreement to use the custom fonts).

### PR DESCRIPTION
**Pull request by dersim-davaod**
Resolves #71

### CODE REVIEW

**Review by marinkman — CHANGES_REQUESTED** [review]
I can't build the project.
You need to replace the image here.
Screenshot 2023-05-25 at 10 32 35

**Comment by dersim-davaod** [comment]
@marinkman you're right, the issue should be fixed now.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift [hunk 1]
```diff
@@ -2,28 +2,41 @@ import SwiftUI
...
+                    selectedDate: .now
+                )
             }
-            EstimatedSleepDurationLabelView()
+
+            MultilineCenteredLabelView(
+                text: estimatedSleepText
+            )
         }
-        .padding()
```
Comment (marinkman) [inline Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:21]: Why did you remove the padding here?

**Review by meschya — APPROVED** [review]
LGTM

**Review by marinkman — COMMENTED** [review]
And I think now you need to add space above the `MultilineCenteredLabelView` on the `SleepSetupView` so that it is below.

File: Sleep Tracker/SleepTracker/View/ReusableViews/MultilineCenteredLabelView.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+/// A label that shows a center-aligned multiline bold text.
+struct MultilineCenteredLabelView: View {
+    /// A text to show in the label.
+    let text: String
```
Comment (MikhailKosarev) [inline Sleep Tracker/SleepTracker/View/ReusableViews/MultilineCenteredLabelView.swift:6]: we will change the text when the user sets the alarm. Should it be `@state var`?

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift (hunk 1 above)
Comment (dersim-davaod) [reply Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:21]: > Why did you remove the padding here?

The external view should set up the external paddings, cause' the external view knows more about the layout.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift [hunk 3]
```diff
@@ -2,28 +2,41 @@ import SwiftUI
...
     }
+
+    // MARK: - Private interface
+
+    private let estimatedSleepText = "Predicted sleep duration:\n7h 52m"
```
Comment (MikhailKosarev) [inline Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:33]: should it be `@state var` instead of  `private let` if it changes every time the user sets the alarm time? Or turns off the alarm.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift (hunk 1 above)
Comment (dersim-davaod) [reply Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:21]: > And I think now you need to add space above the MultilineCenteredLabelView on the SleepSetupView so that it is below. 

We can set up the exact spacing later. For now:
- The layout paddings in the Figma vary from screen to screen, so I consider them as reference, not as rules. Do you have a suggestion on how much space we should have here? How it is differ from device to device?
- It is not so important for release, I believe the whole UX is still clear.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift [hunk 4]
```diff
@@ -2,28 +2,41 @@ import SwiftUI
 
 /// A view shows the options that the user can set before go to bed.
```
Comment (MikhailKosarev) [inline Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:3]: As we discussed in Slack it's not a good description. Maybe it would be better to write it this way?

```suggestion
// A view where the user can set the alarm and choose specific time when it goes off.
```

**Review by MikhailKosarev — CHANGES_REQUESTED** [review]
Good job! Consider some improvements/answer questions:

File: Sleep Tracker/SleepTracker/View/ReusableViews/MultilineCenteredLabelView.swift (hunk 2 above)
Comment (dersim-davaod) [reply Sleep Tracker/SleepTracker/View/ReusableViews/MultilineCenteredLabelView.swift:6]: For now we're not doing this. We can make it `@State` in the corresponding PR.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift (hunk 3 above)
Comment (dersim-davaod) [reply Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:33]: For now we're not doing this.

File: Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift (hunk 4 above)
Comment (dersim-davaod) [reply Sleep Tracker/SleepTracker/View/SleepSetupView/SleepSetupView.swift:3]: Agree, missed the point, thanks!

**Review by marinkman — APPROVED** [review]
LGTM

**Review by MikhailKosarev — APPROVED** [review]
LGTM
